### DISCUSSION TITLE
Data Importer not supports Variants

### DISCUSSION THREAD

**Discussion by mohammed-mahmoud-scandi** [pimcore/pimcore#14878] [Q&A / Support]
I have configured my data importer in the following way to support adding variants to their parents
![image](https://user-images.githubusercontent.com/121216666/230358783-3c534209-3136-4327-9e5a-1812439a647e.png)
and it's working well, but child objects added as objects not variants 
![image](https://user-images.githubusercontent.com/121216666/230358920-af5f7337-f6fd-4037-adb4-4f9eb76bfd4d.png)
though my class supports adding variants
![image](https://user-images.githubusercontent.com/121216666/230359071-c4f15353-23fb-433e-a4f9-309ea1f79b7c.png)
and I can add variants manually 
![image](https://user-images.githubusercontent.com/121216666/230362633-335a898a-834f-4c64-9cd2-6c12cde802c5.png)
Though I went through this  issue which should resolve my issue https://github.com/pimcore/data-importer/issues/180
but it seems issue still here

**Comment by fmcsecurity** [comment]
90% there, simply set the object type to "variant" as shown in the photo below.

It will then be created as a variant of the parent, instead of a standalone object.

<img width="754" alt="variant-setup" src="https://github.com/pimcore/pimcore/assets/133785026/21b7d958-c99f-423a-a046-550c617b95dd">

**Comment by fashxp** [comment]
not sure if @fmcsecurity solution is supported by default. but with next version to come, it will be possible, see also https://github.com/pimcore/data-importer/pull/375
